lib/src/linden/xcolors.dart

```diff
@@ -108,6 +108,9 @@ class XColors {
   Color get shadowTransparent =>
       _theme(bright: _black.withAlpha(65), dark: _black.withAlpha(0));
 
+  Color get shadowTransparentLigher =>
+      _theme(bright: _black.withAlpha(24), dark: _black.withAlpha(0));
```

Comment:
❓ maybe for dark we can just use `transparent`? 

and for bright (as far as I remember) - there should be new color from new schema (it should be one of the gray color). The main goal is to use one of those colors and do not create a custom variations (this also applies to the UI team - use one of those colors)

Reply:
take a look at `NewXColors` 😉 (for now in the app we use them)